x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/AutoFollowCoordinator.java

```diff
@@ -239,20 +259,25 @@ public void clusterChanged(ClusterChangedEvent event) {
         private final String remoteCluster;
         private final Consumer<List<AutoFollowResult>> statsUpdater;
         private final Supplier<ClusterState> followerClusterStateSupplier;
+        private final LongSupplier relativeTimeProvider;
 
+        private volatile long lastAutoFollowTime = -1;
```

Comment:
Can we add the time unit to the variable names (relativeTimeProvider and lastAutoFollowTime)?